framework/Optimizers/RavenSampled.py

```diff
@@ -599,6 +604,10 @@ def _updateSolutionExport(self, traj, rlz, acceptable, rejectReason):
     if self._minMax == 'max':
       objValue *= -1
     toExport[self._objectiveVar] = objValue
+    # check for anything else that solution export wants that rlz might provide
+    for var in self._solutionExport.getVars():
+      if var not in toExport and var in rlz:
+        toExport[var] = rlz[var]
```

Comment:
Should these lines be moved to line 614 (just after the updates of toExport dict)?

Reply:
sure, moving.